control_panel_api/aws.py

```diff
@@ -1,18 +1,20 @@
 import json
 
-import boto3
+from django.conf import settings
+
+aws_api_client = settings.AWS_API_CLIENT_HANDLER
 
 
 def create_bucket(name, region, acl='private'):
-    boto3.client('s3').create_bucket(
+    aws_api_client('s3').create_bucket(
```

Comment:
You know that interestingly while working on the app IAM policy creation I had a similar idea?

I still wonder if there is a way to globally mock boto3 client to return a mock while testing (again, as a way to protect ourself by "accidentally" use `boto.client()` instead of this).

Reply:
Yeh this is what it's doing!

Reply:
Kind of. Here I'm explicitly calling `aws_api_client()` - what happens if a new dev comes in and by accident/distraction calls `boto3.client()`? 

What I'm trying to say is that ideally (if possible!) we'd want our code to not change and still call `boto3.client()` but it would return a mock if running the tests/`ENV` is `test`...it's a subtle difference.

Reply:
`boto3` isn't imported, so calling `boto3.client` would fail. And we'd probably catch that in a code review?

Reply:
Yeh I think reading the code it's clear this is the paradigm, and this is the aws file where it's all located. You're talking about monkey patching it, which then adds a confusing layer of magic, how do they know what they're doing etc.

Reply:
@andyhd I was talking in general, not specifically in this file. Anyway, if you think this is enough I'm happy with the explicit import of this specific client.